migration.md

```diff
@@ -0,0 +1,63 @@
+# Questions
+
+This document needs to be cleaned but I thought this might be better than an infinate issue thread 😉
+
+## General
+- I see a lot of `fs` usage in the [with-mdx-remote example](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote)
+  should this worry me? (as far as I know this is not a browser-supported package and only works locally)
+  
+- I cannot even run the [with-mdx-remote example](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote) example what should I do?
+  
+  My steps & issue:
+  ```
+  gh clone vercel/next.js   # uses the github cli
+  cd next.js/examples/with-mdx-remote
+  yarn  # install dependencies
+  yarn add --dev eslint eslint-config-next  # to be able to use nextjs 11
+  yarn build  # 👈 Exception time
+  ```
+  
+  The exception:
+  ```
+  yarn run v1.22.4
+  $ next build
+  info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
+  info  - Checking validity of types
+  
+  > Build error occurred
+  Error: Failed to load plugin 'jest' declared in '..\..\.eslintrc.json#overrides[0]': Cannot find module 'eslint-plugin-jest'
```

Comment:
This has nothing to do with `next-mdx-remote`, as much as it is unfortunate that it happened